src/NonPhpFile/NonPhpFileProcessor.php

```diff
@@ -4,23 +4,17 @@
 
 namespace Rector\Core\NonPhpFile;
 
-use Rector\ChangesReporting\Application\ErrorAndDiffCollector;
-use Rector\Core\Configuration\Configuration;
 use Rector\Core\Configuration\RenamedClassesDataCollector;
+use Rector\Core\Contract\Processor\NonPhpFileProcessorInterface;
+use Rector\Core\ValueObject\StaticNonPhpFileSuffixes;
 use Rector\PSR4\Collector\RenamedClassesCollector;
 use Symplify\SmartFileSystem\SmartFileInfo;
-use Symplify\SmartFileSystem\SmartFileSystem;
 
 /**
  * @see \Rector\Tests\Renaming\Rector\Name\RenameClassRector\RenameNonPhpTest
  */
-final class NonPhpFileProcessor
+final class NonPhpFileProcessor implements NonPhpFileProcessorInterface
```

Comment:
This service should be probably the one to put those 2 methods in.

It should accept array of `NonPhpFileProcessorInterface[]` services in constructor.

Reply:
I see. But renaming the old class seems hard, because it is a container for everything. So what should be the name of it?

Reply:
Not sure, as I don't know the wholesome architecture yet. Any ideas?

Reply:
No ideas, i just call it NonPhpFileProcessorService for now. Because i don´t really know what is the NonPhpFileProcessor is for.